app/src/ui/history/commit-list.tsx

```diff
@@ -8,26 +8,52 @@ import { IGitHubUser } from '../../lib/databases'
 const RowHeight = 50
 
 interface ICommitListProps {
-  readonly onCommitChanged: (commit: Commit) => void
-  readonly onScroll: (start: number, end: number) => void
-  readonly onRevertCommit: (commit: Commit) => void
-  readonly onViewCommitOnGitHub: (sha: string) => void
-  readonly repository: Repository
-  readonly history: ReadonlyArray<string>
-  readonly commits: Map<string, Commit>
+  /** The GitHub repository associated with this commit (if found) */
+  readonly gitHubRepository: GitHubRepository | null
+
+  /** The list of commits to display, in order. */
+  readonly commits: ReadonlyArray<string>
```

Comment:
Previously this was `history`, and I wanted to make this a bit more generic for use with the Compare experience.